### DISCUSSION TITLE
Import Metric units when there are multiple options?

### DISCUSSION THREAD

**Discussion by AnderssonPeter** [TandoorRecipes/recipes#3040] [Q&A] [answered]
Hi is there someway to import metric units when both exist in a recipe?
Example https://www.justonecookbook.com/simple-chicken-curry/
![image](https://github.com/TandoorRecipes/recipes/assets/1308309/764fe5f0-4655-4cc1-9b93-16a2e20ea6fc)

I don't know what platform they use but it fairly common that there is a button to change to metric units.

**Comment by vabene1111** [comment, marked as answer]
interesting. The problem is that the importer just requests the URL and gets a schema from the page. If the schema has some machine readable distinction between metric and imperial it could be done but in this case the schema just contains `2  onions ((large; 1.5 lb, 670 g))",` whcih the parser cannot handle. 

There are plans to improve the parser in the future using possible LLM or other parsing techniques, maybe those could have a favor for the measurement system but currently that wont be possible.

**Reply by AnderssonPeter** [reply]
Thanks for your response!
